### PR TITLE
Make it clear I am not the only author

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Tests passed, only hvvm failed for unrelated reasons

File: composer.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
 			"name": "Jeroen De Dauw",
 			"email": "jeroendedauw@gmail.com",
 			"homepage": "https://www.entropywins.wtf/"
+		},
+		{
+			"name": "Wikidata Team"
```
Comment (thiemowmde) [inline composer.json]: The existing pattern in other codebases goes like this:
```JSON
{
	"name": "The Wikidata team",
	"role": "Developer"
}
```
Can we follow this here to stay consistent?

Comment (JeroenDeDauw) [reply composer.json]: Totally. Please use the edit button

Comment (Ladsgroup) [reply composer.json]: Should we fix issues in your pull request? Is it the new process of code review?

**Review by thiemowmde — APPROVED** [review]

**Pull request merged**